pkg/translator/api/ingress.go

```diff
@@ -61,7 +82,17 @@ func (o *IngressEdgeLBPoolSpec) Validate(obj *extsv1beta1.Ingress) error {
 	}
 	// Validate that the HTTP port is valid.
 	if err := validation.IsValidPortNum(int(*o.Frontends.HTTP.Port)); err != nil {
-		return fmt.Errorf("%d is not a valid port number", *o.Frontends.HTTP.Port)
+		return fmt.Errorf("%d is not a valid HTTP port number", *o.Frontends.HTTP.Port)
```

Comment:
Can we add the field path to the error please? And use the original message `%d is not a valid port number`.

Reply:
Perhaps specifying what the range is would be good too.